beetsplug/web/templates/index.html

```diff
@@ -80,6 +80,12 @@
                         <a target="_blank" href="http://musicbrainz.org/recording/<%= mb_trackid %>">view</a>
                     </dd>
                 <% } %>
+                <% if (art) { %>
```

Comment:
Hmm, you're right—not quite. The trouble is that `art` isn't an attribute on individual items because album art is associated per *album*. We'd need to check whether the item is associated with an album object that has an `artpath` property.